server/channels/app/plugin_install.go

```diff
@@ -220,6 +221,17 @@ func (ch *Channels) installPlugin(bundle, signature io.ReadSeeker, installationS
 		logger.Warn("Failed to notify plugin status changed", mlog.Err(err))
 	}
 
+	ch.cfgSvc.UpdateConfig(func(cfg *model.Config) {
+		if _, ok := cfg.PluginSettings.Plugins[manifest.Id]; !ok {
```

Comment:
I think this code should be part of `Channels.installExtractedPlugin`. Otherwise, the plugin will be enabled before its configuration is ready. That might lead to race conditions.